scripts/olm-e2e.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -e
+
+# olm-e2e.sh orchestrates deployent of the OLM manifests and images from this
```

Comment:
It's worth pointing out that the community-operators CI does all this (albeit with the end goal of running operator-sdk scorecard). Please take a look for inspiration (main script): https://github.com/operator-framework/community-operators/blob/master/scripts/ci/test-operator